ui/packages/catalog-ui-search/src/main/webapp/component/query-add/query-add.view.js

```diff
@@ -224,13 +249,46 @@ module.exports = Marionette.LayoutView.extend({
     },
     saveTemplateToBackend: function() {
         let loadingView = new LoadingView();
+        let _this = this;
+        let _user = user;
         $.ajax({
             url: '/search/catalog/internal/forms/query',
             data: JSON.stringify(this.getQueryAsQueryTemplate()),
             method: 'PUT',
             contentType: 'application/json',
             customErrorHandling: true
         })
+        .done((data, textStatus, jqxhr) => {
+            announcement.announce({
+                title: 'Saved!',
+                message: 'Search form has been saved.',
+                type: 'success'
+            });
+
+            let queryTemplate = _this.getQueryAsQueryTemplate();
+            let sorts = queryTemplate['querySettings'] && queryTemplate['querySettings'].sorts;
```

Comment:
✏️ We should really lean towards calling the explicit Backbone `.get(...)` methods and such when accessing attributes